nixos/modules/services/web-apps/guacamole.nix

```diff
@@ -0,0 +1,136 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  cfg = config.services.guacamole;
+in
+{
+  options = {
+    services.guacamole = {
+      enable = lib.mkEnableOption (lib.mdDoc "Apache Guacamole");
+
+      package = lib.mkPackageOptionMD pkgs "guacamole-server" { };
+      clientPackage = lib.mkPackageOptionMD pkgs "guacamole-client" { };
+
+      extraEnvironment = lib.mkOption {
+        type = lib.types.listOf lib.types.str;
+        default = [ ];
+        example = [ "ENVIRONMENT=production" ];
+        description = lib.mdDoc "Environment Variables to pass to guacd";
+      };
+
+      baseDir = lib.mkOption {
+        type = lib.types.path;
+        default = "/etc/guacamole";
+        description = lib.mdDoc ''
+          Location of $GUACAMOLE_HOME
+        '';
+      };
+
+      host = lib.mkOption {
+        default = "127.0.0.1";
```

Comment:
Looks like this is what `guacd` requires. I haven't tried with an IPV6, maybe it works.

What do you have in mind?

Reply:
Since it can support both, I don't see the problem indicating an IPV4 address. IPV6 is definitely great but as long as it won't be widely spread, I don't think it would be a good idea to advertise it by default.